sw/airborne/firmwares/rotorcraft/stabilization/stabilization_indi.c

```diff
@@ -350,9 +353,9 @@ static void stabilization_indi_calc_cmd(struct Int32Quat *att_err, bool rate_con
 
   struct FloatRates rate_ref;
   if (rate_control) { //Check if we are running the rate controller
-    rate_ref.p = (float)radio_control.values[RADIO_ROLL]  / MAX_PPRZ * STABILIZATION_INDI_MAX_RATE;
-    rate_ref.q = (float)radio_control.values[RADIO_PITCH] / MAX_PPRZ * STABILIZATION_INDI_MAX_RATE;
-    rate_ref.r = (float)radio_control.values[RADIO_YAW]   / MAX_PPRZ * STABILIZATION_INDI_MAX_RATE;
+    rate_ref.p = stabilization_rate_sp.p;
```

Comment:
The stabilization_rate_sp struct is now defined in the stabilization_rate_indi c file, but if you do not build with rate control this will not be defined, leading to a compile error. The best thing would be to move that definition to this c file and the stabilization_ind_simple.c file. The declaration can remain in the stabilization_indi_header file.